src/Routes/PrivateRoutes.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Kanvas\Routes;
+
+use Baka\Router\Route;
+
+/**
+ * Default Kanvas Private Routes Class
+ */
+class PrivateRoutes
+{
+    private $privateRoutes = [
+        Route::crud('/users')->controller('ApiController')->action('transporter')->notVia('post'),
+        Route::crud('/companies')->controller('ApiController')->action('transporter'),
```

Comment:
@rwhite27 I would leave the ApiController as something the user can pass to the fucntion to define the routes, allowing them to add it to any controller they want